### PR TITLE
disable all addons upon reset to have clean uninstall and doesnt hang.

### PR DESCRIPTION
**Pull request by balchua**
This PR will first disable all addons to cleanly remove addon resources created.
Prevents namespace stuck in terminating phase, when addons are cleanly disabled and removed.

### CODE REVIEW

**Comment by balchua** [comment]
address this issue: https://github.com/ubuntu/microk8s/issues/877

File: microk8s-resources/wrappers/microk8s-reset.wrapper [hunk 1]
```diff
@@ -17,6 +17,15 @@ Options:
 EOF
 }
 
+disable_addons() {
+  echo "Disabling all addons."
+  actions="$(get_all_addons)"
+  for action in $actions; do
+    echo "Disabling addon : $action"
+    microk8s.disable $action > /dev/null 2>&1 || true
```
Comment (ktsakalozos) [inline microk8s-resources/wrappers/microk8s-reset.wrapper]: After enabling storage its disable process gets stuck in:
```
Remove PVC storage at /var/snap/microk8s/common/default-storage ? (Y/N)
```

Having a non interactive enable/disable mode for all addons would have made our life easier.

Comment (balchua) [reply microk8s-resources/wrappers/microk8s-reset.wrapper]: I will work on this one. Thanks for reviewing.

Comment (balchua) [reply microk8s-resources/wrappers/microk8s-reset.wrapper]: @ktsakalozos will it be fine to force an argument to `microk8s.disable storage:destroy-storage`  without specifying the argument will assume to keep the storage. wdyt?
thanks.

Comment (ktsakalozos) [reply microk8s-resources/wrappers/microk8s-reset.wrapper]: Sorry I missed that @balchua. Yes this is a reasonable approach.

**Comment by balchua** [comment]
I have to update `pip3` in the `travis.yaml` file `pip3 install --upgrade pip`, otherwise it kept failing when installing `pytest`

**Comment by balchua** [comment]
Ive modified `microk8s.disable storage` to include `destroy-storage` args.

If arg is not specified, it will not delete the filesystem.

`microk8s.disable storage:destroy-storage`

File: microk8s-resources/wrappers/microk8s-reset.wrapper (hunk 1 above)
Comment (ktsakalozos) [reply microk8s-resources/wrappers/microk8s-reset.wrapper]: @balchua I was wrong to suggest we could change the UI of `microk8s.disable storage` mid-cycle. On the last commit you can see I maintained the old behavior of asking for confirmation in deleteing the storage *if* no arguments were passed.

**Comment by ktsakalozos** [comment]
LGTM +1 thank you @balchua

**Pull request merged**